doc/api/fs.md

```diff
@@ -5141,6 +5146,10 @@ On Windows, opening an existing hidden file using the `'w'` flag (either
 through `fs.open()` or `fs.writeFile()` or `fsPromises.open()`) will fail with
 `EPERM`. Existing hidden files can be opened for writing with the `'r+'` flag.
 
+All of the flags can be prepended with `'m'` to use a memory file mapping for
+accessing the file (for example `'max+'`). This is only available on Windows,
+on other operative systems `'m'` is ignored.
```

Comment:
```suggestion
on other operating systems `'m'` is ignored.
```

Reply:
I guess while we're at it, let's add a comma and make this its own sentence:

```suggestion
On other operating systems, `'m'` is ignored.
```